lib/lita/handlers/jira_issues.rb

```diff
@@ -80,6 +82,26 @@ def priority(data)
       def issue_link(key)
         "\n#{config.url}/browse/#{key}"
       end
+
+      def silenced?(key)
+
+        if config.issue_ttl == 0
+          log.debug("JiraIssues: issue_ttl is set to 0, will post every matched issue to chat")
+          return false
+        end
+
+        current_ttl = redis.ttl(key).to_i
```

Comment:
I think `to_i` errs on the side of caution and will return a 0

```
[1] pry(main)> 'hi there'.to_i
=> 0
```

I guess that's fine, it'll just set an expiry key. Leave it as is :+1: